src/test/java/com/microsoft/graph/http/MockConnection.java

```diff
@@ -18,6 +18,10 @@
     private final ITestConnectionData mData;
     private HashMap<String, String> mHeaders = new HashMap<>();
     private Boolean mFollowRedirects;
+    /**
+     * The encoding type for getBytes
+     */
+    static final String JSON_ENCODING = "UTF-8";
```

Comment:
This time we are fine, because you are encoding a JsonResponse

Reply:
Ok, Keeping this as JSON_ENCODING.